code/game/atoms.dm

```diff
@@ -2056,49 +2056,51 @@
 				| (held_item && SEND_SIGNAL(held_item, COMSIG_ITEM_REQUESTING_CONTEXT_FOR_TARGET, context, src, user))
 
 			if (contextual_screentip_returns & CONTEXTUAL_SCREENTIP_SET)
+				var/screentip_images = active_hud.screentip_images
 				// LMB and RMB on one line...
-				var/lmb_text = (SCREENTIP_CONTEXT_LMB in context) ? "[SCREENTIP_CONTEXT_LMB]: [context[SCREENTIP_CONTEXT_LMB]]" : ""
-				var/rmb_text = (SCREENTIP_CONTEXT_RMB in context) ? "[SCREENTIP_CONTEXT_RMB]: [context[SCREENTIP_CONTEXT_RMB]]" : ""
+				var/lmb_text = build_context(context, SCREENTIP_CONTEXT_LMB, screentip_images)
```

Comment:
Let's test merge this for a bit because this proc is flaming hot and I am worried about the effects of so many proc calls being added to it, CC @LemonInTheDark

Reply:
Like it is very possible that we just stick to monochrome images to make this simpler even

Reply:
this is not giving me great vibes I'll admit

Reply:
will check in detail tomorrow/day after